src/nupic/bindings/algorithms.i

```diff
@@ -1672,7 +1672,19 @@ inline PyObject* generate2DGaussianSample(nupic::UInt32 nrows, nupic::UInt32 nco
 //--------------------------------------------------------------------------------
 // Temporal Memory
 //--------------------------------------------------------------------------------
-%include <nupic/algorithms/TemporalMemory.hpp>
+%inline %{
+  template <typename IntType>
+  inline PyObject* vectorToList(const vector<IntType> &cellIdxs)
+  {
+    PyObject *list = PyList_New(cellIdxs.size());
```

Comment:
Yes, I believe so. `PyList_New` creates a new list with ref count 1. When the function returns, it transfers the ownership to the Python caller. If the Python caller then discards that list, it decrements the ref count is to 0, at which point the list will be freed.

Similarly, `PyList_SET_ITEM()` takes ownership of the reference returned by `PyInt_FromLong`.